packages/traffic_light/src/traffic_light_node.py

```diff
@@ -116,6 +116,18 @@ def change_direction(self, event):
             self.traffic_cycle.shutdown()
             self.traffic_cycle = rospy.Timer(rospy.Duration(cycle_duration), self.change_direction)
 
+    def to_led_order(self, unordered_list):
+        """Change ordering from successive (0,1,2,3,4) to the one expected by the led emitter (0,4,1,3,2)
+
+            Args:
+                unordered_list (:obj:`list`): List to be ordered.
+            Returns:
+                ordered_list (:obj: `list`): Permutated list of length ~number_leds.
```

Comment:
I think it should be:
```:obj:`list`: Permutated list of length ~number_leds```

Reply:
What do I need to change?

Reply:
line 125 to :obj:`list`: Permutated list of length ~number_leds